tutorials/grid_coordinates.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _grid_coordinates:
+
+Gridding coordinates
+====================
+
+Grid coordinates in Verde are used to create points on a regularly spaced grid
+that are then used in the spline method to interpolate between sample data
+points. The grid can be specified either by the number of points in each
+dimension (the *shape*) or by the grid node spacing.
+
+Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`
+function. The function uses :func:`numpy.linspace` and
+:func:`numpy.meshgrid` to create a set of regulary spaced points in both the
+west-east and south-north directions to create a two-dimensional spatial grid.
+To this end, the function will adjust either the region bounds or the spacing
+between grid nodes.
+
+First let's create a region that is 1000 units west-east and 1000 units south-

```

Comment:
The same for this paragraph:
```
First let's create a region that is 1000 units west-east and 1000 units south-
north, and we will set an initial spacing to 100 units.
```

Reply:
Also simplified

Reply:
The line break at the `-` makes results in a space when rendered to html

Reply:
I would move this paragraph to after the `import` block so that the text and variables/functions are closer together.